packages/web-components/src/components/lightbox-media-viewer/lightbox-video-player-composite.ts

```diff
@@ -35,6 +35,20 @@ class DDSLightboxVideoPlayerComposite extends ModalRenderMixin(DDSVideoPlayerCom
    */
   private _hCloseModal: Handle | null = null;
 
+  /**
+   * Handles aria state depending on the modal's state.
+   */
+  private _handleAriaState = () => {
+    const iFrame = this._videoPlayer?.querySelector('iframe');
+
+    // Handles edge case where screen reader still reads video title within iFrame
+    if (this.open) {
+      window.frames[`${iFrame?.id}`].document.querySelector('.topBarContainer').removeAttribute('aria-hidden');
```

Comment:
Also I suggest putting this logic in `try` ... `catch` block. Cross-frame DOM access is often restricted:

```suggestion
      iFrame.contentWindow.document.querySelector('.topBarContainer').removeAttribute('aria-hidden');
```